recipes/r-pim/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set version = '2.0.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pim
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/pim_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/pim/pim_{{ version }}.tar.gz
+  sha256: 1195dbdbd67348dfef4b6fc34fcec643da685ebe58d34bbe049ab121aca9944f
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-bb
+    - r-nleqslv
+  run:
+    - r-base
+    - r-bb
+    - r-nleqslv
+
+test:
+  commands:
+    - $R -e "library('pim')"           # [not win]
+    - "\"%R%\" -e \"library('pim')\""  # [win]
+
+about:
+  home: https://github.com/CenterForStatistics-UGent/pim
+  license: GPL-2.0-only
+  summary: 'Fit a probabilistic index model as described in Thas et al, 2012: <doi:10.1111/j.1467-9868.2011.01020.x>.
+    The interface to the modeling function has changed in this new version. The old
+    version is still available at R-Forge.'
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```

With `GPL (>= 2)` I usually include both, which is consistent with how CRAN website displays it.

```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```